sample/Sample/Program.cs

```diff
@@ -1,12 +1,18 @@
-﻿using System.Linq;
+﻿using System.Collections.Generic;
+using System.Linq;
 using Serilog;
 using Serilog.Sinks.Splunk;
 
 namespace Sample
 {
     public class Program
     {
-        public static string EventCollectorToken = "2B94855F-1184-46F7-BFF1-56A3112F627E";
+        const string SPLUNK_FULL_ENDPOINT = "http://ws2012-devops:8088/services/collector";
+            //  Patrik local vm machine   
+
+        const string SPLUNK_ENDPOINT = "http://ws2012-devops:8088"; //  Patrik local vm machine    
```

Comment:
@patriklindstrom FYI your VM info is here.  Would the Sample work better with some args? e.g. HEC token and host?